client/src/components/ResponsiveAppBar/ResponsiveAppBar.js

```diff
@@ -0,0 +1,164 @@
+import * as React from 'react';
+import AppBar from '@mui/material/AppBar';
```

Comment:
These are repeated imports that will be removed once you have imported the components and icons separately
- TODO: Remove imports from line 6 to 18 once you have imported the components and icons separately